sound/soc/intel/boards/sof_sdw.c

```diff
@@ -146,8 +146,8 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 			DMI_MATCH(DMI_PRODUCT_NAME, "Volteer"),
 		},
 		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
-					SOF_SDW_PCH_DMIC |
-					SOF_SDW_FOUR_SPK),
+					SOF_SDW_PCH_DMIC | SOF_SDW_FOUR_SPK |
+					SOF_SSP_PORT(SOF_I2S_SSP2)),
```

Comment:
that probably works but that macro was intended for amplifiers.
We should probably have a different macro to support the case of SDW headset + I2S amplifier + BT offload. not sure if it makes sense but it's better to plan ahead.

Reply:
Ack, a new macro to distinguish I2S used for BT-offload purpose is the way to go.